generation/tasks/PlotNPCorr.py

```diff
@@ -298,15 +279,15 @@ def output(self):
 
 
     def run(self):
-        PlotNPCorrSummary.check_outdir(self.output())
+        check_outdir(self.output())
         print("=======================================================")
         print("Starting NP-factor summary plotting with YODA")
         print("=======================================================")
 
         inputs = dict()
-        print("\n\n{}\n\n".format(self.input()))
-        inputs["ratio"] = self.localize_input(self.input()["ratio"])
-        inputs["Fits"] = self.localize_input(self.input()["Fits"]["single"])
+        logger.debug("\n\nInput:\n{}\n\n".format(self.input()))
```

Comment:
Isn't this logging message now printed in the localize input function? So this line can be removed fully.

Reply:
done